ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/ImmutableDB/StateMachine.hs

```diff
@@ -16,6 +16,30 @@
 {-# LANGUAGE UndecidableInstances #-}
 
 {-# OPTIONS_GHC -Wno-orphans      #-}
+-- | Model-based tests for the immutable DB.
+--
+-- This is the main test for the immutable DB. As in any model based, we have a
+-- set of commands, which in this case corresponds to things like:
+--
+-- * Read a block, or information about a block, from the DB
+-- * Append a block to the database
+-- * Stream blocks from the DB
+-- * etc.
```

Comment:
i particularly dislike the trailing `etc` but perhaps it is just a matter of taste

Reply:
Makes sense. I removed it and added a reference to 'Cmd'.